package.json

```
@@ -20,11 +21,15 @@
     "url": "https://github.com/tc39/tc39.github.io/issues"
   },
   "homepage": "https://tc39.es/",
+  "engines": {
+    "node": "^14.8"
+  },
   "devDependencies": {
     "prettier": "^1.17.1",
     "stylelint": "^10.0.1",
     "stylelint-config-prettier": "^5.2.0",
     "stylelint-config-standard": "^18.3.0",
-    "stylelint-order": "^3.0.0"
+    "stylelint-order": "^3.0.0",
+    "vnu-jar": "20.6.30"
```

I'm big on pinning dependencies, but this could also be a caret range.

```suggestion
    "vnu-jar": "^20.6.30"
```